src/page-modules/departures/__tests__/departure.test.tsx

```diff
@@ -47,67 +28,6 @@ describe('departure page', function () {
     };
 
     const output = render(<DeparturesPage {...initialProps} />);
-    expect(await output.findAllByText(/^Result/)).toHaveLength(2);
-  });
-
-  it('Should return props from getServerSideProps', async () => {
```

Comment:
I removed this test for now, and didn't replace it, since I removed all of the data fetching. We can add it later when we fetch data for the departures list/ map.

Reply:
Instead of removing the data fetching and tests, maybe we could include the `getServerSideProps` function from earlier using the query parameter? 👍

Reply:
Can do that.